core/capabilities/targets/write_target.go

```diff
@@ -131,8 +131,14 @@ func (cap *WriteTarget) Execute(ctx context.Context, request capabilities.Capabi
 		WorkflowExecutionID: rawExecutionID,
 		ReportId:            inputs.ID,
 	}
+
```

Comment:
Bind above is failing, double check all tests and usage of `Bind()` here and in chainlink-feeds and chainlink-solana too